contracts/Token.sol

```diff
@@ -104,9 +113,10 @@ contract GolemNetworkToken {
         Migrate(msg.sender, migrationAgent, _value);
     }
 
-    // Set address of migration target contract and enable migration process.
-    // Required state: Operational Normal
-    // State transition: -> Operational Migration
+    /// @dev Set address of migration target contract and enable migration process.
```

Comment:
Use `@notice` for this sentence despite this function being available only for migration master. Please, limit line length to 80.